### PR TITLE
Cwagent

### PR DESCRIPTION
**Pull request by jlsclaranet**
Hello,

I've just added detectors for AWS based on metrics reported by cloudwatch agent.
This takes into account disk usage, ram usage, cpu usage and heartbeat.
I've run a test on client if you ever want to have a look (I put low values in order to make it sound / alarm).
Ask me the client name if needed
Best regards,

### CODE REVIEW

File: modules/integration_aws-cwagent/conf/readme.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+documentations:
+
```
Comment (Shr3ps) [inline modules/integration_aws-cwagent/conf/readme.yaml:2]: there is really no documentation?

**Review by Shr3ps — CHANGES_REQUESTED** [review]

File: docs/severity.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 - [integration_aws-apigateway](#integration_aws-apigateway)
 - [integration_aws-backup](#integration_aws-backup)
 - [integration_aws-beanstalk](#integration_aws-beanstalk)
+- [integration_aws-cwagent](#integration_aws-cwagent)
```
[the remark this answers is not in the document]
Comment (jlsclaranet) [reply docs/severity.md:14]: Hello,

Thanks for your feedback.
The cloudwatch agent metrics are in cloudwatch and are brought up by the AWS integration. I didn't change anything in the initial setup. 
The documentation indicates: SOURCE_TYPE="integration" # or "internal", "integration", "organisation", "otel-collector".

I suppose I'm in the right place, aren't I?

All the best,

File: modules/integration_aws-cwagent/conf/01-mem_used_percent.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Example
+module: AWS CWAgent
+name: "Utilisation de la mémoire"
```
[the remark this answers is not in the document]
Comment (jlsclaranet) [reply modules/integration_aws-cwagent/conf/01-mem_used_percent.yaml]: I'm sorry I didn't think of English, I just pushed a button. Thanks for your feedback

File: modules/integration_aws-cwagent/conf/readme.yaml (hunk 1 above)
Comment (jlsclaranet) [reply modules/integration_aws-cwagent/conf/readme.yaml]: Thank you for your comment.
I hadn't seen that readme, the main one being generated.
I've compared it with the other detectors and I think it's compliant now. Is it good for you?

**Comment by jlsclaranet** [comment]
And I forgot to mention that the client hadn't pushed the tags yet, so I removed the filters manually in the interface to be able to test the detectors.

File: docs/severity.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jlsclaranet) [reply docs/severity.md]: Hello, I've renamed. Is it OK now plz ?

**Review by Shr3ps — APPROVED** [review]

**Pull request merged**